scripts/entrypoint_openshift.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+export USER_NAME=geoserveruser
+export HOME=/home/geoserveruser
+if ! whoami &> /dev/null; then
+  if [ -w /etc/passwd ]; then
```

Comment:
Should also test if the passwd contains the `$USER_NAME` entry.
In some cases, our user restart the container without deleting it. In those case, the passwd file will have duplicate entries.

Reply:
ahh yes, that's indeed something we definately need to change. will check

Reply:
actually the test for the $USER_NAME entry is already there, when $USER_NAME is not in /etc/passwd the "whoami" command fails and that enters the if.

Reply:
Oh yes, you are right. It's not that obvious to me.